### ISSUE TITLE
great code!  can you impl macd &  candlestick?

### ISSUE THREAD

**Issue by sunwangme** [wcharczuk/go-chart#3]
rt

**Comment by wcharczuk** [comment]
hey! macd for sure at some point; i need to implement a histogram drawer, and once that's done it'll just mean tweaking the ema series to take a window size and macd should be doable.

candlestick will require a bit more thought as we'd need (4) data-points to power it, the open, close high and low, which i haven't figured out a way to codify in the model yet (we really only have y ranges as []float64). 

in short, macd yes, candlestick will take a bit more work and will probably happen later

**Comment by wcharczuk** [comment]
i have a first stab at macd series in the repo now; take a a peek. one thing i'm struggling with is how to verify that my implementation is producing the correct values (short of making an excel model and working backwards). i wouldn't use this yet until i get some better tests in there, but it should be good for at least running through a sanity check.

**Comment by wcharczuk** [comment]
ok, i made an excel model and created test cases to verify, should be good to go on macd

**Comment by sunwangme** [comment]
good news

**Comment by zfsamzfsam** [comment]
+1

**Comment by dfontana** [comment]
Do you have an example of how this was intended to be implemented? MACDSeries, MACDLineSeries, and MACDSignalSeries all do not implement the Series interface, so they cannot be passed to a Chart's Series field like other charts may be implemented
